src/vsphere_cpi/lib/cloud/vsphere/cloud.rb

```diff
@@ -41,6 +44,44 @@ def enable_telemetry
       other_client.logout rescue nil
     end
 
+    def enable_extension
```

Comment:
Should this method be named create_cpi_extension instead as you are either returning and existing/creating a new extension right?

Reply:
yep, I can rename that.